GCD.py

```diff
@@ -0,0 +1,53 @@
+#define a function, that given two positive integers, m and n, the function will return their greatest common divisor,GCD
+import random
+def gcd(m,n):
+    if m==n:
+        return m
+    elif m<n:
+        m,n=n,m
+    if m%n==0:
+        return n
+    else:
+        i=n-1
+        while i>1:
+            if m%i==0 and n%i==0:

```

Comment:
alternatively, this can be re-written as:

```python
       for i in range(n-1, 1, -1)：
            if m%i==0 and n%i==0:
                return i
        return 1
```